### PR TITLE
irssi: update to 1.4.4

### PR DESCRIPTION
**Pull request by dynasticorpheus**
Compile tested: Ubuntu 23.04 x64 / OpenWrt SNAPSHOT @ 6cf27094e9209250dbd45f8b042530c3b23f0a42
Run tested: Linksys WRT32X / OpenWrt SNAPSHOT r24002-e3559fb445

Description:

- update to 1.4.4
- switch to meson
- remove obsolete patch

Signed-off-by: Fabian Lipken [dynasticorpheus@gmail.com](mailto:dynasticorpheus@gmail.com)

### CODE REVIEW

**Comment by dynasticorpheus** [comment]
@BKPepe Tagging you as you seem involved with previous commits for this package.

**Comment by BKPepe** [comment]
> @BKPepe Tagging you as you seem involved with previous commits for this package.

While looking at https://github.com/openwrt/packages/commits/master/net/irssi/Makefile, I think you want to ping @tripolar , who is the maintainer of this package.

**Comment by dynasticorpheus** [comment]
@tripolar

File: net/irssi/Makefile [hunk 1]
```diff
@@ -8,23 +8,23 @@
...
-PKG_HASH:=a647bfefed14d2221fa77b6edac594934dc672c4a560417b1abcbbc6b88d769f
+PKG_HASH:=fefe9ec8c7b1475449945c934a2360ab12693454892be47a6d288c63eb107ead
 
 PKG_LICENSE:=GPL-2.0-or-later
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:irssi:irssi
 
-PKG_FIXUP:=autoreconf
-PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=python3/host
```
Comment (neheb) [inline net/irssi/Makefile:22]: what?

Comment (dynasticorpheus) [reply net/irssi/Makefile]: Thanks, removed.

**Comment by neheb** [comment]
add -Dwith-otr=no .

**Pull request merged**